core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -878,8 +878,8 @@ static unsigned int get_physical_addr_size_bits(void)
 	 * 0b101      48 bits, 256TB.
 	 * 0b110      52 bits, 4PB (not supported)
 	 */
-
-	COMPILE_TIME_ASSERT(CFG_CORE_ARM64_PA_BITS >= 32);
+	static_assert(CFG_CORE_ARM64_PA_BITS >= 32);
+	static_assert(CFG_CORE_ARM64_PA_BITS <= 52);
```

Comment:
This function never returns `TCR_PS_BITS_4PB`

Reply:
Thanks, I'll update.